pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -165,8 +171,8 @@ type MonitorContainer struct {
 }
 
 // +k8s:openapi-gen=true
-// TidbClusterRef reference to a TidbCluster
-type TidbClusterRef struct {
+// ClusterRef reference to a TidbCluster
```

Comment:
Is it OK to define `type ClusterRef TidbClusterRef` and we can use `ClusterRef` in the new code without touching the old code using `TidbClusterRef`?